xbmc/cores/VideoPlayer/VideoRenderers/ColorManager.h

```diff
@@ -0,0 +1,173 @@
+/*
+ *      Copyright (C) 2016 Lauri Mylläri
+ *      http://kodi.org
+ *
+ *  This Program is free software; you can redistribute it and/or modify
+ *  it under the terms of the GNU General Public License as published by
+ *  the Free Software Foundation; either version 2, or (at your option)
+ *  any later version.
+ *
+ *  This Program is distributed in the hope that it will be useful,
+ *  but WITHOUT ANY WARRANTY; without even the implied warranty of
+ *  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ *  GNU General Public License for more details.
+ *
+ *  You should have received a copy of the GNU General Public License
+ *  along with XBMC; see the file COPYING.  If not, see
+ *  <http://www.gnu.org/licenses/>.
+ *
+ */
+
+#pragma once
+
+#if defined(HAVE_LCMS2)
+#include "lcms2.h"
+#endif
+
+#include <string>
+
+enum CMS_MODE
+{
+  CMS_MODE_3DLUT,
+  CMS_MODE_PROFILE,
+  CMS_MODE_COUNT
+};
+
+enum CMS_WHITEPOINT
+{
+  CMS_WHITEPOINT_D65,
+  CMS_WHITEPOINT_D93,
+  CMS_WHITEPOINT_COUNT
+};
+
+enum CMS_PRIMARIES
+{
+  CMS_PRIMARIES_AUTO,
+  CMS_PRIMARIES_BT709,    // HDTV
+  CMS_PRIMARIES_170M,     // SDTV
+  CMS_PRIMARIES_BT470M,   // old NTSC (1953)
+  CMS_PRIMARIES_BT470BG,  // old PAL/SECAM (1975)
+  CMS_PRIMARIES_240M,     // old HDTV (1988)
+  CMS_PRIMARIES_COUNT
+};
+
+enum CMS_TRC_TYPE
+{
+  CMS_TRC_BT1886,
+  CMS_TRC_INPUT_OFFSET,
+  CMS_TRC_OUTPUT_OFFSET,
+  CMS_TRC_ABSOLUTE,
+  CMS_TRC_COUNT
+};
+
+class CColorManager
+{
+public:
+  /*!
+   \brief Access the global color management system
+   \return the global instance
+   */
+  static CColorManager& Get();
```

Comment:
no singletons, this is an anti pattern. embed the object where you want to use it.